automatminer/automl/adaptors.py

```diff
@@ -118,9 +118,9 @@ def fit(self, df, target, **fit_kwargs):
         self._features = df.drop(columns=target).columns.tolist()
         self._ml_data = {"X": X, "y": y}
         self.fitted_target = target
-        self.logger.info("TPOT fitting started.")
```

Comment:
Why are there these logger changes happening here? In this PR we shouldn't do this